src/utils/formatWithPrettierEslint.ts

```diff
@@ -0,0 +1,10 @@
+import format from 'prettier-eslint'
+import { SourceFile } from 'ts-morph'
+
+export function formatWithPrettierEslint(sourceFile: SourceFile): SourceFile {
+    const filePath = sourceFile.getFilePath().toString()
+
+    sourceFile.replaceWithText(format({ text: sourceFile.getFullText(), filePath }))
```

Comment:
Didn't know that prettier has sync API. It actually makes sense since it's changing files but still interesting.